sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,60 @@
 Version history
 ===============
 
+6.12.0 (2022 February)
+----------------------
+
+File format fixes and improvements:
+
+* Lambert Instruments FLIM
+   - added support for MSB 12 to 16 bit conversion (thanks to Shaquille Louisa - Lambert Instruments)
+
+* Olympus cellSens VSI
+   - corrected tile handling to respect tile origin metadata
+   - pixel data will now only be read from ets files with the prefix frame_
```

Comment:
As noted in https://github.com/openmicroscopy/data_repo_config/pull/561, this should include `funded by a partnership between Glencoe Software and Evident Scientific`.